pkg/client/client.go

```diff
@@ -929,54 +930,54 @@ func (c Client) validatePrometheusResource(ctx context.Context, prom types.Names
 	avail, err := getMonitoringCondition(p.Status.Conditions, monv1.Available)
 	if err != nil {
 		err = fmt.Errorf("prometheus: %w", err)
```

Comment:
```suggestion
		err = fmt.Errorf("get prometheus status: %w", err)
```

Reply:
I've removed the prefix since the error is unlikely to happen in real life (e.g. it'd be a bug in the code or in the Prometheus operator not populating the conditions properly).